slider/src/components/SliderPlugin.vue

```diff
@@ -20,7 +20,12 @@ export default {
     defaultValue() {
       return this.options.defaultValue
         ? Number.parseFloat(this.options.defaultValue)
-        : 0
+        : this.minValue
+    },
+    minValue() {
+      return this.options.minValue
+        ? Number.parseFloat(this.options.minValue)
+        : undefined
```

Comment:
```suggestion
      : 0
```

On a side note; it could be convenient to have default values for `minValue` and `maxValue`